src/headers.js

```diff
@@ -24,7 +24,7 @@ function validateValue(value) {
 }
 
 /**
- * @typedef {Headers | Record<string, string> | Iterable<readonly [string, string]> | Iterable<string>[]} HeadersInit
+ * @typedef {Headers | Record<string, string> | Iterable<readonly [string, string]> | Iterable<string[]>} HeadersInit
```

Comment:
Please check this: https://github.com/node-fetch/node-fetch/pull/841#issuecomment-634764277. It's iterable of string, not string array.

Reply:
I just fixed it with `Iterable<Iterable<string>>` that is a more robust and compatible method.